server/src/metadata/field-metadata/field-metadata.entity.ts

```diff
@@ -35,11 +34,14 @@ export type FieldMetadataTargetColumnMap = {
 })
 @QueryOptions({
   defaultResultSize: 10,
-  maxResultsSize: 100,
   disableFilter: true,
   disableSort: true,
 })
-@Unique('IndexOnNameAndWorkspaceIdUnique', ['name', 'objectId', 'workspaceId'])
+@Unique('IndexOnNameObjectIdAndWorkspaceIdUnique', [
```

Comment:
Name was wrong, the migration AddUniqueConstraintsOnFieldObjectMetadata1697630766924 contains the right name but for some reason I pushed the entity with a different name. It should be aligned now with the existing migration. (which is also why we don't see an index change in the migration introduced in this PR)